docs/upcoming_changes/9051.cuda.rst

```diff
@@ -0,0 +1,8 @@
+
+Add CUDA SASS CFG Support
+=========================
+
+This PR This PR adds support for getting the SASS CFG in dot language format. 
+It adds an inspect_sass_cfg method to CUDADispatcher and the -cfg flag to
+the nvdisasm command line tool.
```

Comment:
For rst I think we can highlight these as code (and add brackets to the function name) with:

```suggestion
It adds an ``inspect_sass_cfg()`` method to CUDADispatcher and the ``-cfg``
flag to the nvdisasm command line tool.
```